python/foglamp/common/configuration_manager.py

```diff
@@ -635,9 +638,9 @@ def get_entry_val(k):
                 if cat_item is not None:
                     category_value = await self._read_category_val(category_name)
                     if category_value is not None:
-                        self._cacheManager.update(category_name, category_value)
-                        self._cacheManager.cache[category_name]['value'].update({item_name: cat_item})
                         category_value = self._handle_script_type(category_name, category_value)
+                        self._cacheManager.update(category_name, category_value)
```

Comment:
as above.

> this will force here to set display_name = category_name in cache, we should create a ticket to check self._cacheManager.update(category_name, category_value) calls and handle this; And also a story to handle category-> description as well. And add them here with FIXME and TODO

Reply:
FOGL-3246